conformance/utils/kubernetes/helpers.go

```diff
@@ -321,16 +321,7 @@ func MeshNamespacesMustBeReady(t *testing.T, c client.Client, timeoutConfig conf
 	require.NoErrorf(t, waitErr, "error waiting for %s namespaces to be ready", strings.Join(namespaces, ", "))
 }
 
-// GatewayAndHTTPRoutesMustBeAccepted waits until:
-//  1. The specified Gateway has an IP address assigned to it.
-//  2. The route has a ParentRef referring to the Gateway.
-//  3. All the gateway's listeners have the following conditions set to true:
-//     - ListenerConditionResolvedRefs
-//     - ListenerConditionAccepted
-//     - ListenerConditionProgrammed
-//
-// The test will fail if these conditions are not met before the timeouts.
```

Comment:
It seems like it would be useful to keep some variation of this comment here

Reply:
Done.